src/Api/Controllers/CiphersController.cs

```diff
@@ -76,7 +76,7 @@ public async Task<CipherMiniResponseModel> GetAdmin(string id)
         {
             var cipher = await _cipherRepository.GetOrganizationDetailsByIdAsync(new Guid(id));
             if (cipher == null || !cipher.OrganizationId.HasValue ||
-                !await _currentContext.ManageAllCollections(cipher.OrganizationId.Value))
+                !await _currentContext.ViewAllCollections(cipher.OrganizationId.Value))
```

Comment:
I can see an argument for `edit` permissions here, but I went with `view` since I feel that delete rights should give you the ability to see all collections a cipher is in.